device-types/SNR/S2995G-24TX-POE.yaml

```diff
@@ -0,0 +1,189 @@
+---
+manufacturer: SNR
+model: S2995G-24TX-POE
+slug: snr-s2995g-24tx-poe
+part_number: SNR-S2995G-24TX-POE
+u_height: 1.0
+is_full_depth: false
+weight: 6
+weight_unit: kg
+airflow: right-to-left
+comments: '[SNR-S2995G-24TX-POE](https://snr.systems/ru/catalog/setevoe-oborudovanie/kommutatory-dostupa/snr-s2995gtx/snr-s2995g-24tx-poe)'
+console-ports:
+  - name: con0
+    label: CONSOLE
+    type: rj-45
+power-ports:
+  - name: power AC
+    type: iec-60320-c14
+    maximum_draw: 400
+power-outlets:
+  - name: usb
```

Comment:
USB-a ports as power-outlets is not something currently being modeled in this repo